docs/nlu/components.rst

```diff
@@ -946,6 +946,9 @@ EntitySynonymMapper
         pipeline:
         - name: "EntitySynonymMapper"
 
+    .. note::
+        When using the EntitySynonymMapper as part of an NLU pipeline, it will need to be placed below the Entity Extractor in the configuration file.
```

Comment:
```suggestion

        When using the ``EntitySynonymMapper`` as part of an NLU pipeline, it will need to be placed
        below any entity extractors in the configuration file.
```